src/auth/basic/UserRequest.cc

```diff
@@ -56,17 +56,13 @@ Auth::Basic::UserRequest::authenticate(HttpRequest *, ConnStateData *, Http::Hdr
         return;
 
     /* are we about to recheck the credentials externally? */
-    if ((user()->expiretime + static_cast<Auth::Basic::Config*>(Auth::SchemeConfig::Find("basic"))->credentialsTTL) <= squid_curtime) {
+    if (user()->expiretime <= squid_curtime) {
         debugs(29, 4, "credentials expired - rechecking");
```

Comment:
> Currently only used for the Basic, Negotiate, and NTLM scheme authentication helpers

Please warn (in cache.log) when the new annotation is received while using other schemes (where it is still not supported).

Reply:
Done in [47d75af](https://github.com/squid-cache/squid/pull/1662/commits/47d75af1ef87e7183626edb2b13923a3fd955cd1)